packages/styles/extends.import.styl

```diff
@@ -0,0 +1,15 @@
+.header-font
+  font-family 'Oswald', sans-serif
+
+.body-font
+  font-family 'Open Sans', sans-serif
+
+.icon
+  font-family FontAwesome
+  -webkit-font-smoothing antialiased
```

Comment:
Good call...nib takes care of it but it doesn't add to mozilla prefix which is fine with me.